configure.ac

```diff
@@ -571,6 +571,53 @@ if test "X$xslt_config" != "Xno" ; then
   fi
 fi
 
+libxslt_source=no
+if test "X$XSLT_CONFIG" != "X"; then
+  XSLT_CFLAGS=`$XSLT_CONFIG --cflags`
+  XSLT_LIBS=`$XSLT_CONFIG --libs`
+
+  CPPFLAGS="$XSLT_CFLAGS $CPPFLAGS"
+  LIBS="$LIBS $XSLT_LIBS"
+
+  AC_CHECK_FUNC(xsltSaveResultToString, have_xsltSaveResultToString=yes, have_xsltSaveResultToString=no)
+  AC_MSG_CHECKING(for libxslt via xslt-config)
+  if test $have_xsltSaveResultToString = yes; then
+    have_libxslt=1
+    LIBXSLT_VERSION=`$XSLT_CONFIG --version`
+    libxslt_version_dec=`echo $LIBXSLT_VERSION | $AWK -F. '{printf("%d\n", 10000*$1 + 100*$2 + $3)};'`
+    libxslt_min_version_dec=`echo $libxslt_min_version | $AWK -F. '{printf("%d\n", 10000*$1 + 100*$2 + 3)};'`
+    AC_MSG_RESULT(yes - version $LIBXSLT_VERSION)
+    if test $libxslt_version_dec -lt $libxslt_min_version_dec; then
+       AC_MSG_WARN(Using libxslt $LIBXSLT_VERSION is unsupported - $libxslt_min_version or newer required.)
+       have_libxslt=0
+    fi
+  fi
+
+  AC_CHECK_FUNC(xsltInit)
+
+  AC_CHECK_HEADERS(libxslt/xslt.h)
+  if test "$ac_cv_header_libxslt_xslt_h" = no ; then
+    AC_MSG_WARN(libxslt library found but not headers - disabling)
+    have_libxslt_lib=0
+    have_libxslt=0
+  fi
+  CPPFLAGS="$oCPPFLAGS"
+  LIBS="$oLIBS"
+else
+  PKG_CHECK_MODULES([XSLT], [libxslt > $libxslt_min_version], [
+    LIBXSLT_VERSION=`$PKG_CONFIG libxslt --modversion`
+    libxslt_source="pkg-config"
+  ], [:])
+  AC_MSG_CHECKING(for libxslt via pkg-config)
+  if test "$libxslt_source" != "no"; then
+    have_libxslt=1
+    AC_MSG_RESULT(yes - $LIBXSLT_VERSION)
```

Comment:
The if for libxslt_source will always fail so it is not needed.  612-618 can be replaced with
```
have_libxslt_=0
AC_MSG_RESULT(no - not found)
```

Reply:
I still feel much of configure.ac is overly complicated, particularly with libxml and libxslt. The minimum libxslt version has been around for 18 years. So, we could easily cut the function and header checks. If you're interested, that would be a separate PR.